authentication-guides/azure-function-app/README.md

```diff
@@ -0,0 +1,355 @@
+---
+name: Azure Function Apps   # The name of the guide
+description: Connect Creator Studio to Azure Function Apps   # Brief description of the guide
+time_in_minutes: 30   # Time in minutes to complete the guide
```

Comment:
I would quote this as 60 minutes to set up local Python and all the stuff around this

Reply:
Done